ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapLayout.tsx

```diff
@@ -30,6 +31,8 @@ export const AtlasmapLayout: FunctionComponent<IAtlasmapLayoutProps> = ({
 }) => {
   const { pending, isEditingMapping } = useAtlasmapUI();
 
+  const toggleFreeView = useCanvasViewOptionsContext().toggleFreeView;
```

Comment:
`const { toggleFreeView } = useCanvasViewOptionsContext();` maybe?

Reply:
That actually worked. Didn't realize it could figure out just by the name I wanted the method rather than the flag.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring